_data/toc/frontend-developer-guide.yml

```diff
@@ -99,8 +99,8 @@ pages:
         - label: Customize email templates
           url: /frontend-dev-guide/templates/template-email.html
 
-        - label: Templates XSS security
-          url: /frontend-dev-guide/templates/template-security.html
+        - label: XSS prevention strategies
```

Comment:
This entry already exists in the PHP Developer Guide and shouldn't be duplicated in the Frontend Developer Guide. You should remove this entry if your intention is to eliminate the frontend version of the topic.

Reply:
I have removed this entry. Thanks.